ifsbench/data/namelisthandler.py

```diff
@@ -164,7 +166,86 @@ def execute(self, wdir, **kwargs):
         debug(f"Modify namelist {input_path}.")
         namelist = f90nml.read(input_path)
 
+        namelist.uppercase = True
+        namelist.end_comma = True
+
         for override in self.overrides:
             override.apply(namelist)
 
         namelist.write(output_path, force=True)
+
+class SanitiseMode(Enum):
```

Comment:
If this is serialised, I think you need (str, Enum) to correctly print the strings in a nice format.